src/olympia/constants/activity.py

```diff
@@ -894,6 +895,36 @@ class REPLY_RATING(_LOG):
     store_ip = True
 
 
+class CLEAR_NEEDS_HUMAN_REVIEW(_LOG):
+    id = 176
+    format = '{version} no longer flagged for human review.'
+    short = 'Needs Human Review cleared'
+    admin_event = True
+    review_queue = True
+    reviewer_review_action = True
+    hide_developer = True
+
+
+class CLEAR_PENDING_REJECTION(_LOG):
+    id = 177
+    format = _('{version} pending rejection cleared.')
+    short = _('Pending rejection cleared')
```

Comment:
Are these two localized for a reason?  (when the other actions you've added aren't)

Reply:
If that action appears in devhub, it would be good to have it localized. If not, we don't need localization.

Reply:
@wagnerand should it show in devhub, do you think?

Reply:
Are we showing a pending (delayed) rejection in the devhub? If so, where?
If we were to show the clearing, where would we show it?

Reply:
Yes, we expose the activity log for `REJECT_VERSION_DELAYED` ("{addon} {version} reject scheduled.") in the activity section in devhub.  This action would be exposed in the same place.

p.s. having looked at the action wording, maybe it should be `'{version} scheduled rejection cleared.` to match? `</bikeshed>`

Reply:
If we expose `REJECT_VERSION_DELAYED` and we persist that even beyond `CLEAR_PENDING_REJECTION` (which @eviljeff confirmed in chat), then we should also expose `CLEAR_PENDING_REJECTION` to show the full trail of that activity to the developer. And as such, localizing the activity would be good.